app/controllers/api/v1/book_suggestion_controller.rb

```diff
@@ -0,0 +1,37 @@
+module Api
+  module V1
+    class BookSuggestionController < ApplicationController
```

Comment:
You should use ApiController

Reply:
If I use ApiController it would not comply with what the card asks for because it asks to create a suggestion without authorization, in ApiController this line is `before_action :authenticate_user` and when using ApiController it would be demanding that you log in to be able to create a suggestion

Reply:
you should use `skip_before_action: :authenticate_user, only: :create` its more clear